packages/ledger-amino/src/launchpadledger.ts

```diff
@@ -166,6 +167,17 @@ export class LaunchpadLedger {
     await this.verifyCosmosAppIsOpen();
   }
 
+  public async verifyAddress(hdPath: HdPath): Promise<AddressAndPublicKeyResponse | ErrorResponse> {
+    await this.verifyDeviceIsReady();
+    assert(this.app, `${this.ledgerAppName} Ledger App is not connected`);
```

Comment:
`this.app` is set in the constructor and is non-optional. So we don't need the check here.

Also after `await this.verifyDeviceIsReady();` multiple things are checked already based on `this.app`.